blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfig.java

```diff
@@ -44,13 +45,13 @@ private String createConfig() {
                     .key("security")
                     .object()
                         .key("enabled").value(jenkins.isUseSecurity())
-                        .key("authorizationStrategy")
-                        .object()
+                        .key("authorizationStrategy").object()
                             .key("allowAnonymousRead").value(allowAnonymousRead)
+                            .endObject()
+                        .key("enableJWT").value(FEATURE_JWT)
```

Comment:
it is in a system property as a top level thing I think, this is just in the client side config object.